specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/stable/2019-06-01/machineLearningServices.json

```diff
@@ -1811,6 +1811,37 @@
       ],
       "x-ms-discriminator-value": "DataLakeAnalytics"
     },
+    "OnPrem": {
+      "description": "A OnPrem compute.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/Compute"
+        },
+        {
+          "type": "object",
+          "properties": {
+            "properties": {
```

Comment:
Should it be flattened instead of the double properties?

Reply:
Hi @nschonni , I am following the same existing format such as those compute type in the json: "AmlCompute", "DataLakeAnalytics", "Databricks", "DataFactory", "HDInsight"